File: ArtSiteApi/ArtSite.Data/Models/PersonalityModel.cs
namespace ArtSite.Data.Models; 

public class PersonalityModel: BaseModel {
    public string PersonalityInfo { get; set; }
    public string Traits { get; set; }
    public string Flaws { get; set; }
    public byte Introverted { get; set; }
    public byte Intuitive { get; set; }
    public byte Thinking { get; set; }
    public byte Judging { get; set; }
    public byte Assertive { get; set; }
}

File: ArtSiteWeb/artsite-web/src/network/models/personalityModel.ts
import { BaseModel } from "./baseModel";
import { BaseNameModel } from "./baseNameModel";

export interface PersonalityModel extends BaseModel {
    personalityInfo: string;
    introverted: number;
    intuitiv: number;
    thinking: number;
    judging: number;
    assertive: number;
    traits: BaseNameModel[];
    flaws: BaseNameModel[];
}
